web_ir_actions_act_multi/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://www.gnu.org/graphics/lgplv3-147x51.png
+   :target: https://www.gnu.org/licenses/lgpl-3.0.en.html
+   :alt: License: LGPL-v3
+
+=================
+Web Actions Multi
+=================
+
+This module provides a way to trigger more than one action on ActionManager
+
+Usage
+=====
+
+To use this functionality you need to return following action with list of actions to execute:
+
+.. code-block:: python
```

Comment:
Hi zaoral, what's normal block of code? Can you please provide me an example. Thanks :)

Reply:
> Hi @zaoral, what's normal block of code? Can you please provide me an example. Thanks :)

`code-block` is used by sphinx, is not standard rst directive. In rst you can create role to simulate this. you can find an example [here](https://stackoverflow.com/questions/10870719/inline-code-highlighting-in-restructuredtext)

But that is not the important issue, the reason why I was asking you to change it was because it was throwing a WARNING in the server log that was making this PR a fake "red runbot". But as far as I can see now odoo has integrated the rst package needed for support code-block directve since the last time you push. So now  you do not need to change the directive you can use ..code-block

Reply:
Great, thanks. Can you then please give thumbs up so we can have two positive reviews and merge this PR with 11.0 branch ...